components/confirmationPopup/ConfirmationPopup.styles.js

```diff
@@ -0,0 +1,129 @@
+import styled from 'styled-components';
```

Comment:
I believe there is an overlay on the entire screen and the scrolling is stopped when the pop-up comes up, I did something similar on my PR for the Home Page if you want to take a look